### ISSUE TITLE
behave setup is accessing test and dev dbs

### ISSUE THREAD

**Issue by odoric27** [devops-s17-payments/payments#158]
not sure how it's happening or how to fix yet, but i noticed that the "given the following payments" data, which is added to the db via the PaymentService static method, is somehow hitting both the test and dev databases.

the steps of each scenario are only making changes to the test data, but the initial seeding of data is hitting both. if anyone has any thoughts or encounters a fix while working on bdd stuff, let me know.

**Comment by odoric27** [comment]
Update. Found a workaround which is included in new PR ( #159 ), but here's some more details on the problem:

I think the use of the staticmethod `load_sample` was somehow updating both the dev db and test db. I tried making it an instance method, including it within the context of the bdd feature, but to no avail.

After some frustrated tinkering, I gave up on this approach and elected to do a POST within the context of the app. Perhaps not an ideal solution to use a feature to seed data that we're ultimately testing with some of that data, but for the moment, things do seem to work.

one caveat with this approach was that somehow the dev db was still being touch, but ONLY for the first POST. So payment with id 1 gets sent to dev, but everything else gets sent to test. very bizarre.

i initially tried to find a way to cut off access to dev db, but the way our app sets up is such that the db gets initialized prior to behave doing anything. so while we designate the test db for the test_client in `before_all`, the dev db is still connected underneath it all so some requests (like the load_sample) that weren't wholly contained within the behave context were hitting both dbs.

the ultimate workaround for this was send out a blank POST prior to seeding the sample data. through all of my tests thus far, this seems to work, but if somehow wants to take a crack at a  better solution, have at it.

otherwise, i'll close this out when #159 gets merged
